docs/tests.md

```diff
@@ -86,15 +95,12 @@ With the Simpletest UI apparently going away in Drupal 8.2 (cf [#2566767],
 The typical full command to run tests looks like this (`\` is to avoid too long a line):
 
     MONGODB_URI=mongodb://somemongohost:27017 \
-    SIMPLETEST_DB=mysql://someuser:somepassword@localhost/somedatabase \
-    phpunit -c core/phpunit.xml.dist
+    PHPUNIT_OPTS="-c phpunit.xml -v --debug --coverage-clover=modules/contrib/$MODULE_NAME/$COVERAGE_FILE"
 
 * Optional: `MONGODB_URI` points to a working MongoDB instance. This variable is optional:
   if it is not provided, the tests will default to `mongodb://localhost:27017`.
-* Required: `SIMPLETEST_DB` is the standard Drupal 8 variable needed to run tests that
-  can need the database service.
 
-Both variables can be set in the `core/phpunit.xml` custom configuration file.
+Above variable can be set in the `core/phpunit.xml` custom configuration file.
```

Comment:
Due to the additional variables made necessary by the Functional test, we have a plural again, sorry.